libs/libuwsc/Makefile

```diff
@@ -8,13 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libuwsc
-PKG_VERSION:=2.0.5
-PKG_RELEASE:=1
+PKG_VERSION:=3.0.4
+PKG_RELEASE:=2
```

Comment:
please reset release number to 1 on version bump

Reply:
Done.